tests/testthat/test-quality.R

```diff
@@ -0,0 +1,30 @@
+context("quality")
+
+test_that("model quality metrics work as expected", {
+  mod <- lm(mpg ~ wt, data = mtcars)
+
+  expect_equal(mse(mod, mtcars), 8.697561, tolerance = 0.0000001)
```

Comment:
Having very fined grained floating point numbers in tests makes me very nervous — these are likely to fail for uninteresting reasons (like the compiler optimises slightly differently) — and I have no idea where these numbers come from.